routes/gym.js

```diff
@@ -20,6 +20,78 @@ router.route('/').get(async (req, res) => {
   }
 });
 
+//Wild Card Search bar
```

Comment:
What is the change here?

Reply:
I switched the order of the routes since if you put router.route('/:id') first it will think 'search', 'manage', and 'add' are gym Id's and will go to that route.

Reply:
so router.route('/manage') was never getting hit and would go to router.route('/:id') instead

Reply:
I do not think the order matters, since we are not doing a partial match. Did you meet this problem while you test?

Reply:
Yes I did.